neuralteleportation/experiments/weights_histogram.py

```diff
@@ -124,9 +130,17 @@ def main():
                                  output_dir=args.output_dir, save_format=args.save_format,
                                  xlim=args.xlim, ylim_max=args.ylim_max)
 
+    # Plot histogram of weights with same distribution
+    # sampled from the cumulative distribution
+    model = simulate_teleportation_distribution(deepcopy(net), config=config)
```

Comment:
Since you're using `config` only here, I'd move init the object right here with something like:
```suggestion
    model = simulate_teleportation_distribution(deepcopy(net), config=DistributionTeleportationTrainingConfig())
```
and get rid of line 115 altogether. It's not a good practice to create an object far away from where it's used, since it only makes the code harder to read.